pkg/controller/configmap_watcher.go

```diff
@@ -36,16 +53,17 @@ func NewConfigMapWatcher(cfg *rest.Config) (ConfigMapWatcher, error) {
 	}, nil
 }
 
-// GetStaticDisabledChecks returns an existent ConfigMap data's disabled checks, if they exist
-func (cmw *ConfigMapWatcher) GetStaticDisabledChecks(ctx context.Context) ([]string, error) {
+// GetStaticKubelinterConfig returns the ConfigMap's checks configuration
+func (cmw *ConfigMapWatcher) GetStaticKubelinterConfig(ctx context.Context) (config.Config, error) {
+	var cfg config.Config
+
 	cm, err := cmw.clientset.CoreV1().
 		ConfigMaps(configMapNamespace).Get(ctx, configMapName, v1.GetOptions{})
 	if err != nil {
-		return []string{}, fmt.Errorf("gathering starting configmap: %w", err)
+		return cfg, fmt.Errorf("gathering starting configmap: %w", err)
```

Comment:
nit. This error messages seems to me a bit strange to be honest. What about something like getting initial configuration or something like that?

Reply:
Also should we return empty `Config` struct in this case?

Reply:
> nit. This error messages seems to me a bit strange to be honest. What about something like getting initial configuration or something like that?

Taking advantage on before comment, I'll revamp new log messages as well.

> Also should we return empty Config struct in this case?

That config returned is not being used anywhere. I usually send the most information I have to possible debugging in the future if something goes wrong. If you think is a better idea to send an empty object, I don't have a strong opinion on this.